application/services/learning_analytics/get_dashboard_log.rb

```diff
@@ -0,0 +1,138 @@
+# frozen_string_literal: true
```

Comment:
This file should use 2 spaces for tabs throughout -- it sometimes uses 2 spaces, and sometimes 4 spaces.